crypto/s2n_dhe.c

```diff
@@ -52,7 +52,7 @@ static const BIGNUM *s2n_get_p_dh_param(struct s2n_dh_params *dh_params)
 #if S2N_OPENSSL_VERSION_AT_LEAST(1, 1, 0)
     DH_get0_pqg(dh_params->dh, &p, NULL, NULL);
 #else
-    p                      = dh_params->dh->p;
```

Comment:
Well this is strange. Clearly this was aligned on purpose -- but I don't think its more legible with the alignment.